packages/strapi-deepsearch-service/__tests__/component-utils.test.ts

```diff
@@ -1,22 +1,47 @@
-import { StrapiBuilder } from "strapi-builder";
```

Comment:
Idea of strapi builder was to be able to mock Strapi global object, with adding additional customisable fields like components plugins etc, but in normal situation instance of Strapi from strapi builder should return a clean strapi object (typed) 

I am wondering why do You cause to use it, and create your own instance of strapi, only with components.

in future when `isModelComponent` function will change to use another fields from global object, test will fails and will require to add this fields to your builder Instance

I would like to have 1 package for StrapiBuilder for all packages, not each builder per package

Reply:
I think in this situation is not important because this is used in tests, and any tests should create to own Strapi instance to prevent changes in this global object between tests